packages/types/src/sign-client/client.ts

```diff
@@ -60,9 +60,14 @@ export declare namespace SignClientTypes {
 
   type Metadata = CoreTypes.Metadata;
 
+  type SignConfig = {
+    disableRequestQueue?: boolean;
+  };
```

Comment:
nitpick, optional: shouldn't this be in types?

Reply:
what do you mean? It is in `packages/types/src/sign-client/client.ts`